src/loc.rs

```diff
@@ -22,16 +22,11 @@ pub struct LocDiff {
     file: String,
 }
 
-/// The methods and associated functions on and for a `LocDiff` instance,
-/// respectively.
+/// The methods and associated functions on and for a `LocDiff`, respectively.
 impl LocDiff {
     /// The getter method for the field `file` of the corresponding struct.
-    ///
-    /// Since the struct `LocDiff` defines its field `file` to be private, this
-    /// getter is used in order to operate on this field.  It returns a
-    /// read-only deep copy of the content.
-    pub fn file(&self) -> String {
-        String::from(&self.file)
+    pub fn file(&self) -> &str {
+        &self.file
```

Comment:
This should be squashed with the commit, that introduced this as a function returning `String`.